client/src/main/scala/drt/client/services/handlers/StaffMovementsHandler.scala

```diff
@@ -43,10 +42,14 @@ class StaffMovementsHandler[M](modelRW: ModelRW[M, (Pot[Seq[StaffMovement]], Vie
 
     case GetStaffMovements() =>
       val (_, viewMode) = value
-      log.info(s"Calling getStaffMovements with ${SDate(viewMode.millis).toISOString()}")
+      val maybeMillis = viewMode match {
+        case ViewLive() => None
+        case vm: ViewMode => Option(vm.millis)
+      }
+      log.info(s"Calling getStaffMovements with ${maybeMillis.map(SDate(_).toISOString())}")
```

Comment:
do we want to leave this log in?

Reply:
we have a log line for every call to the api currently, so this is consistent with other calls